test/client.tests.js

```diff
@@ -164,7 +164,7 @@ describe('client initialization', function () {
       const handler = sinon.stub().returns([200]);
       nock('https://op.example.com').get('/foo').reply(handler);
       const client = await getClient({ ...config });
-      await client.requestResource('https://op.example.com/foo');
```

Comment:
I had to add token here, even its value is not important. But it is required now.